### PR TITLE
update tagline and description

### PR DESCRIPTION
**Pull request by orbeckst**
- broaden tag line (not _just_ MD)
- use description from https://numfocus.org/project/mdanalysis

### CODE REVIEW

**Comment by IAlibay** [comment]
We really should tell the numfocus folks change the hyperlink on that page's MDA icon so that it doesn't point to dask 🙄 I'll try to remember next time I email them.

File: _config.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ permalink:        pretty
 # Setup
 title:            MDAnalysis
 name:             MDAnalysis
-tagline:          A Python package for the analysis of molecular dynamics simulations.
-description:      A Python package for the analysis of molecular dynamics simulations.
+tagline:          A Python package for the analysis of molecular simulations.
```
Comment (IAlibay) [inline _config.yml:11]: This is <s>probably</s> definitely my 2 am brain being silly here, but would "molecular simulation data" be better?

Also analysis seems to be short selling MDA (I think nearly 50% of my use nowadays is pre-simulation processing), would "handling and analysis" or something like that be better?

Comment (orbeckst) [reply _config.yml:11]: If you can roll it all (pre and post) into a handy slogan – please do!

Comment (IAlibay) [reply _config.yml:11]: How about?

```suggestion
tagline:          A Python package for the manipulation and analysis of molecular simulations data.
```

File: _config.yml [hunk 2]
```diff
@@ -8,8 +8,8 @@ permalink:        pretty
...
+description:      MDAnalysis is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists.
```
Comment (IAlibay) [inline _config.yml:12]: Maybe this?
```suggestion
description:      MDAnalysis is a Python library which enables the handling and analysis of data from computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
```

**Comment by orbeckst** [comment]
Thanks @IAlibay

**Comment by orbeckst** [comment]
I tried to see if there was an obvious repo in https://github.com/numfocus to raise an issue for the wrong link on https://numfocus.org/project/mdanalysis (MDA icon links to dask) but I'll just email NF.

**Comment by orbeckst** [comment]
Can I merge this?

**Review by IAlibay — APPROVED** [review]

**Comment by IAlibay** [comment]
Apologies @orbeckst I'm not sure why this one took so long, I was convinced we had merged it before sending off the GSOC application :/

**Comment by orbeckst** [comment]
Thanks @IAlibay . I lost track of it and just saw it while preparing for #165

**Comment by orbeckst** [comment]
Oops. I guess I didn't test it locally... it looks awful! Need to do something about it...

**Comment by orbeckst** [comment]
(It doesn't help that my local "show me the built page" ruby mess is broken...)

**Comment by orbeckst** [comment]
I fixed it and directly pushed to master. Sorry, emergency 🆘 fix.

**Pull request merged**